### DISCUSSION TITLE
Adding a WYSIWYG markdown editor

### DISCUSSION THREAD

**Discussion by bzlister** [Flutter-Bounty-Hunters/super_editor#567] [Ideas]
super_editor is *this* close to acheiving a WYSIWYG markdown editor - currently, `super_editor_markdown` exposes a way to render a string of markdown text into a `MutableDocument`, which then can be used with a `DocumentEditor` to make a `SuperEditor`. You can insert new plain text, or edit the text within the existing Markdown syntax, but you cannot add new Markdown syntax.

Are there any plans for this? If not, I would appreciate some advice on getting started on this myself - from reading source code I think I need to find where the raw text is parsed into a `List<DocumentNode>`, and edit it to instead generate the List from `_MarkdownToDocument` as is done in `deserializeMarkdownToDocument`

**Comment by TruFelix** [comment]
Hi, same here, I am dying trying to find a WYSIWYG editor for markdown **&** latex...

really appreciate the project so far, great work and great example of FOSS.

I have the usecase that I want to create something like Obsidian, where markdown has support for injected latex, is there any way I could achieve this with SuperEditor?
Or is something planed, that would allow to do that?
